### DISCUSSION TITLE
.ini file change notice

### DISCUSSION THREAD

**Discussion by Michael21107** [scripthookvdotnet/scripthookvdotnet#1372] [Q&A] [answered]
Will you put a notice in the release description (both nightly and full build) if there's a change to the .ini file so people won't have to reset their settings when unnecessary?

**Comment by kagikn** [comment, marked as answer]
For stable releases, I'll add notes what is changed about the ini in release notes. crosire mentioned how to set ini values in [the release note of v3.0.1](https://github.com/scripthookvdotnet/scripthookvdotnet/releases/tag/v3.0.1). Well, I added notice that a lot of scripts assumes the script folder to be the default one several years later since the release of v3.0.1. I added the example file of the ini starting from 760b75f so you'll be able to know how to set ini values quickly and test them even in nightly builds.

**Reply by Michael21107** [reply]
im just asking if there will be a notice in nightly build’s release cuz thats the closest from now, IF theres a change to the .ini

**Reply by kagikn** [reply]
I’ll update the ini example when I changed something about the ini, I'm kinda too lazy to add notices about the ini to something other than commit messages and the ini example. I'll add notices about how SHVDN changed on parsing the ini (like new options or syntax changes) in release notes of stable releases, however.

**Reply by Michael21107** [reply]
commit messages are the bullet points or details of them, correct?

**Reply by kagikn** [reply]
commit messages are listed in [the commit history](https://github.com/scripthookvdotnet/scripthookvdotnet/commits/main/). Commits contain changes and messages to tell you something such as brief summary of how, what, or why some of the codebase is changed.

**Reply by Michael21107** [reply]
gotcha, thank you sir
